scripts/zkai_native_seq32_attention_mlp_adjacent_label_seed_sweep_gate.py

```
@@ -0,0 +1,680 @@
+        "mutations_rejected": sum(1 for case in cases if case["rejected"]),
+        "mutation_names": list(MUTATION_NAMES),
+        "cases": cases,
+    }
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    ensure_mutation_inventory()
+    validate_payload_core(payload)
+    mutation = payload.get("mutation_result")
+    if not isinstance(mutation, dict):
+        raise AdjacentSeedSweepGateError("mutation_result missing")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)